nuclear_simulator/data_gen/config_engine/initial_conditions/feedwater_conditions_old.py

```
@@ -0,0 +1,468 @@
+        "motor_bearing_wear": [7.8, 7.7, 7.9, 7.6],  # >8.0% triggers motor bearing replacement (100-200 min timing)
+        "pump_bearing_wear": [5.8, 5.7, 5.9, 5.6],   # >6.0% triggers pump bearing replacement (100-200 min timing)
+        "thrust_bearing_wear": [3.8, 3.7, 3.9, 3.6], # >4.0% triggers thrust bearing replacement (100-200 min timing)
+        
+        "description": "Multiple bearing wear scenario - triggers multiple bearing replacements (100-200 min)",
+        "expected_action": "bearing_replacement",
+        "component_id": "all",
+        # Supporting vibration and temperature indicators
+        "vibration_increase": [1.5, 1.3, 1.7, 1.4],  # mm/s increase from wear
+        "oil_temperature": 52.0,  # °C, elevated but below threshold
+        
+        "description": "Bearing wear requiring inspection and cleaning",
```

Ruff (0.11.9)

Dictionary key literal `"description"` repeated

(F601)